src/persist-client/src/internal/metrics.rs

```diff
@@ -1893,6 +1896,41 @@ impl PushdownMetrics {
     }
 }
 
+#[derive(Debug)]
+pub struct BlobMemCache {
+    pub(crate) size_blobs: UIntGauge,
+    pub(crate) size_bytes: UIntGauge,
+    pub(crate) hits_blobs: IntCounter,
+    pub(crate) hits_bytes: IntCounter,
+}
+
+impl BlobMemCache {
+    fn new(registry: &MetricsRegistry) -> Self {
+        BlobMemCache {
+            size_blobs: registry.register(metric!(
+                name: "mz_persist_blob_cache_size_blobs",
+                help: "count of blobs in the cache",
+                const_labels: {"cache" => "mem"},
+            )),
+            size_bytes: registry.register(metric!(
+                name: "mz_persist_blob_cache_size_bytes",
+                help: "total size of blobs in the cache",
+                const_labels: {"cache" => "mem"},
+            )),
+            hits_blobs: registry.register(metric!(
+                name: "mz_persist_blob_cache_hits_blobs",
```

Comment:
nit: elsewhere it's referred to as `BlobMemCache` vs the metric name `blob_cache`

Reply:
yup! did that on purpose to set up a hypothetical disk cache. this has a label of "cache" -> "mem"

Reply:
oh yeah, I totally saw that and forgot it when reading it over a second time